### DISCUSSION TITLE
How to only perform row and not column hiararchecal clustering in Instant Clue?

### DISCUSSION THREAD

**Discussion by stephlhan** [hnolCol/instantclue#82] [Q&A] [answered]
<img width="1339" alt="Screenshot 2023-10-31 at 22 09 56" src="https://github.com/hnolCol/instantclue/assets/149549484/ad48c727-d353-4e9c-9934-a138a3f93cdf">
I want to cluster my row values but still keep my column in sequence, in this case EVKD_0h_1, EVKD_0h_2, etc. But it seems like I can only cluster both axis?

**Comment by hnolCol** [comment, marked as answer]
You can set the columMetric in the settings (bottom-right icon) to None. Then the columns are shown in the order as they are dropped into the Numeric receiver box. 

I've attached a screenshot below. 

<img width="935" alt="image" src="https://github.com/hnolCol/instantclue/assets/24191910/917b1f28-ce85-4e78-8e69-ae2c8feaa9c5">


To center the color map (white == zero) use the "to zero" menu option in the context menu (right-click on the hierarchical clustering icon). 

<img width="361" alt="image" src="https://github.com/hnolCol/instantclue/assets/24191910/4b920235-f0cb-4953-a9dd-bb9efdcdea48">


I will add a context menu option for the hierarchical clustering to enable disable clustering of rows/columns. As well as the centering of the colormap. 
Thank you for the post. A function that is often used is the "To Excel" function that will export the hierarchical clustering into an excel file which is available in the dropdown menu of the graph 
<img width="454" alt="image" src="https://github.com/hnolCol/instantclue/assets/24191910/c3473a4e-668b-45ce-9cfe-06ad745be975">


Cheers

Hendrik

**Reply by stephlhan** [reply]
Wow thank you so much!! 
![Screenshot 2023-11-01 at 09 11 49](https://github.com/hnolCol/instantclue/assets/149549484/1967c571-8193-48ea-8aca-a807de742056)
I have this now, but an add on question: do you know how I can flip the map upside down? i.e. have C(1) on top and C(4) on the bottom, so that the blue block on the left-bottom goes to left_top?

**Reply by hnolCol** [reply]
Hi, 

sadly not possible at the moment but I may add a possibility to reverse the row clustering in the future. Could be a quick and dirty solution to simply reverse the image in a vector graphic software such as adobe illustrator.

**Reply by stephlhan** [reply]
Understood. Thank you so much! Great software☺️

**Reply by hnolCol** [reply]
Thank you and you are welcome. ;) please don’t hesitate to request new features if you are missing some functionality.I hope to be able to invest some time in updating the tutorials.
